src/mixins/raster-layer-line-mixin.js

```diff
@@ -366,6 +368,14 @@ export default function rasterLayerLineMixin(_layer) {
       const filterTransforms = _layer
         .getTransforms(table, filter, globalFilter, state, lastFilteredSize)
         .filter(f => f.type === "filter")
+      const bboxFilter = getContourBoundingBox(
+        state.data[0],
+        chart.map().getBounds()
```

Comment:
This is the exact fn call that's getting messed up with custom dimensions.....sigh